desc/objectives/_neoclassical.py

```diff
@@ -260,3 +266,267 @@ def compute(self, params, constants=None):
+        }
+        self._hyperparameters = {
+            "num_quad": num_quad,
+            "num_pitch": num_pitch,
+            "batch": batch,
+            "num_well": num_well,
+        }
+        self._keys_1dr = ["iota", "iota_r", "min_tz |B|", "max_tz |B|"]
+        if Nemov:
+            self._key = "Gamma_c"
+            self._constants["quad2 x"], self._constants["quad2 w"] = chebgauss2(
+                num_quad
+            )
+        else:
+            self._key = "Gamma_c Velasco"
+
+        super().__init__(
+            things=eq,
+            target=target,
+            bounds=bounds,
+            weight=weight,
+            normalize=normalize,
+            normalize_target=normalize_target,
+            loss_function=loss_function,
+            deriv_mode=deriv_mode,
+            name=name,
+            jac_chunk_size=jac_chunk_size,
+        )
+
+    def build(self, use_jit=True, verbose=1):
+        """Build constant arrays.
+
+        Parameters
+        ----------
+        use_jit : bool, optional
+            Whether to just-in-time compile the objective and derivatives.
+        verbose : int, optional
+            Level of output.
+
+        """
+        eq = self.things[0]
+        self._grid_1dr = LinearGrid(
+            rho=self._constants["rho"], M=eq.M_grid, N=eq.N_grid, NFP=eq.NFP, sym=eq.sym
+        )
+        self._constants["quad x"], self._constants["quad w"] = get_quadrature(
+            leggauss(self._hyperparameters.pop("num_quad")),
+            (automorphism_sin, grad_automorphism_sin),
+        )
+        self._target, self._bounds = _parse_callable_target_bounds(
+            self._target, self._bounds, self._constants["rho"]
+        )
+
+        timer = Timer()
+        if verbose > 0:
+            print("Precomputing transforms")
+        timer.start("Precomputing transforms")
+
+        self._constants["transforms_1dr"] = get_transforms(
+            self._keys_1dr, eq, self._grid_1dr
+        )
+        self._constants["profiles"] = get_profiles(
+            self._keys_1dr + [self._key], eq, self._grid_1dr
+        )
+
+        timer.stop("Precomputing transforms")
+        if verbose > 1:
+            timer.disp("Precomputing transforms")
+
+        super().build(use_jit=use_jit, verbose=verbose)
+
+    def compute(self, params, constants=None):
+        """Compute Γ_c.
+
+        Parameters
+        ----------
+        params : dict
+            Dictionary of equilibrium degrees of freedom, e.g.
+            ``Equilibrium.params_dict``
+        constants : dict
+            Dictionary of constant data, e.g. transforms, profiles etc.
+            Defaults to ``self.constants``.
+
+        Returns
+        -------
+        result : ndarray
+            Γ_c as a function of the flux surface label.
+
+        """
+        if constants is None:
+            constants = self.constants
+        eq = self.things[0]
+        # TODO: compute all deps of gamma here
```

Comment:
issue reference?

Reply:
My comment is opaque, but this is resolved by the fourier bounce stuff. https://github.com/PlasmaControl/DESC/pull/1003/files#r1774068985